base_view_inheritance_extension/README.rst

```diff
@@ -35,8 +35,14 @@ Move an element in the view
 
     <xpath expr="$xpath" position="move" target="$targetxpath" />
 
-This can also be used to wrap some element into another, create the target
-element first, then move the node youwant to wrap there.
+Availables values for position attribute:
+
+* inside (default if omited)
+* after
+* before
+
+This can also be used to wrap some element into another, create the new target
```

Comment:
If this means `simply create a new node around an existing node`, this is already possible with the standard `replace`, [using `$0`](https://www.odoo.com/documentation/11.0/reference/views.html#inheritance-specs).

If this means `create a new node anywhere and move another node in it in a single xpath node`, I think the same logic could be used on the `move` position to do it with a single xpath node.

Reply:
we're in 9, `$0` is a feature from 10 upwards. Counterintuitive as this is, it might indeed make sense to backport this feature instead of cooking up another syntax

Reply:
Oh, you're right, I didn't took attention to the target version :)